source/includes/cpp-library-reference/_simulator.html.md

```diff
@@ -112,33 +109,19 @@ void MySimulator::simulate(const bonsai::InklingMessage& action,
 
 Returns a value indicating whether the simulation is set up to run in predict mode or training mode.
 
-## string objective_name()
-
-```cpp
-void MySimulator::episode_start(const bonsai::InklingMessage& params,
-                                bonsai::InklingMessage& initial_state) {
-    cout << objective_name() << endl;
-    ...
-}
-```
-
-Property accessor that returns the name of the current objective from Inkling.
-The objective may be updated before `episode_start` is called. When running
-for prediction and during start up, objective will return an empty std::string.
-
 ## episode_start(parameters, initial_state)
 
 > Example Inkling:
 
-```inkling
-schema Config
-    UInt8 start_angle
-end
+```inkling2
+type Config {
+    start_angle: Number.UInt8
```

Comment:
does this need a `using Number` statement to work, or is the `Number` namespace automatically "used"?